frontend/src/stores/user.js

```diff
@@ -9,113 +15,120 @@ const DEFAULT_PIN = {
     '3': '',
 };
 
-// local utility helper
-const toStringFromObject = (obj) => Object.values(obj).join('');
-const processPhoneNumber = (phoneNumber) =>
-    phoneNumber.replace(/^0/, '+358').replace(/\s/g, '');
-
-class UserStore {
+class userStore {
     isAuthenticating = false;
-    token = null;
     authenticationFailed = false;
-    phoneNumber = '';
-    phoneNumberIncorrect = false;
     pinCode = DEFAULT_PIN;
-    pinCodeIsSet = false;
-    balance = 100;
+    username = undefined;
```

Comment:
Assigning `undefined` is unnecessary as that is the default value. In my opinion, anything should never be assigned `undefined` value. The semantics just makes no sense there as you are then assigning a value to describe that no value has been set? `null` is instead the correct value for "no value".